plugins/node/opentelemetry-instrumentation-koa/test/utils.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as utils from '../src/utils';
+import * as assert from 'assert';
+import { KoaInstrumentationConfig, KoaLayerType } from '../src/types';
+
+describe('Utils', () => {
+  describe('isLayerIgnored()', () => {
+    it('should not fail with invalid config', () => {
+      assert.doesNotThrow(() => utils.isLayerIgnored(KoaLayerType.MIDDLEWARE));
```

Comment:
I understand the semantics here, but `assert.doesNotThrow` seems inaccurate here. If the function throws, the test would catch it anyway. It would be more expressive to just test the output to be `false` for the cases of invalid config because that's also included in the expectation for the function, isn't it?